Cakes-Co/style.css

```diff
@@ -1 +1,344 @@
+:root {
+  --background: #de9bab;
+  --brand: #36000b;
+  --layout-color: #d1758b;
+  --brand-accent: #263600;
+  --transition-effect: ease-in-out 0.4s;
+}
+
+body {
+  background-color: var(--background);
+  font-family: Arial, Helvetica, sans-serif;
+  color: var(--brand);
+}
+
+a {
+  text-decoration: none;
+}
+
+.cupcake-logo {
+  border-radius: 100%;
+}
+
+header p {
+  color: var(--brand);
+}
+
+main div:nth-child(1) {
+  background-image: url("https://images.unsplash.com/photo-1588195538326-c5b1e9f80a1b?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=3150&q=80");
+  background-size: cover;
+  background-position: center;
+}
+
+.socials-svgs svg {
+  fill: var(--brand);
+}
+
+@media only screen and (min-width: 320px) and (max-width: 480px) {
+  /* GENERAL STYLES */
+
+  * {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+  }
+
+  body {
+    padding-bottom: 4rem;
+  }
+
+  .layout {
+    display: grid;
+    grid-template-rows: 200px 780px 350px 140px;
+    row-gap: 2rem;
+    margin-top: 2rem;
+  }
+
+  nav {
+    display: none;
+  }
+
+  /* HEADER & NAV STYLES - HEADER PICTURE & HAMBURGER MENU */
+
+  header {
+    display: grid;
+    grid-template-columns: 110px 1fr;
+    grid-template-rows: 110px 1fr;
+    row-gap: 1rem;
+    padding: 0 2rem;
+    /* column-gap: 2rem; */
+  }
+
+  .cupcake-logo {
+    width: 100%;
+    height: 100%;
+  }
+
+  header svg {
+    justify-self: end;
+  }
+
+  header p {
+    font-size: 1.2rem;
+    padding-bottom: 1rem;
+    text-align: end;
+    line-height: 0.8;
+    grid-column: 2 / 3;
+  }
+
+  main,
+  nav,
+  content {
+    width: 90%;
+    margin: 0 auto;
+  }
+
+  /* MAIN STYLES - MAIN CAKE IMAGE AND WELCOME TEXT */
+
+  main {
+    display: grid;
+    grid-template-rows: repeat(2, 1fr);
+    row-gap: 2rem;
+  }
+
+  main div:nth-child(1) {
+    grid-column: span 2;
+  }
+
+  main div:nth-child(2) {
+    grid-row: 1 / 2;
+    grid-column: span 2;
+  }
+
+  main div:nth-child(2) p {
+    line-height: 1.4;
+    text-align: justify;
+    font-size: 1.1rem;
+    margin-top: 2rem;
+  }
+
+  main div:nth-child(2) h1 {
+    font-size: 4rem;
+  }
+
+  /* CONTENT STYLES - 4 IMAGES OF CAKES */
+
+  content {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: repeat(2, 10rem);
+    row-gap: 2rem;
+    column-gap: 2rem;
+  }
+
+  /* FOOTER STYLES - A FEW LINKS AND SOCIAL MEDIA LINKS */
+
+  footer div:nth-child(1) {
+    display: flex;
+    gap: 0.4rem;
+    flex-direction: column;
+  }
+
+  footer {
+    margin: 0 auto;
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    align-items: center;
+    column-gap: 2rem;
+    width: 82%;
+  }
+
+  .socials-svgs {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 0.6rem;
+    text-align: end;
+  }
+}
+
+/*  */
+/*  */
+/*  */
+/*  */
+
+@media only screen and (min-width: 767px) and (max-width: 1650px) {
```

Comment:
What caused you to choose these values? What is a "magic number" in CSS?

Reply:
I actually chose the wrong values, as I thought that the max-width was the size of an actual view of a phone from the eye (If that makes sense), but I now realise that its actually to do with the viewport, which is a min-width between 320px — 480px and a max-width between 481px — 768px

Reply:
I've now fixed this issue, as I chose the wrong values